### ISSUE TITLE
Doesn't build properly anymore on release 3080 x64

### ISSUE THREAD

**Issue by ciscoheat** [clemos/haxe-sublime-bundle#209]
Hello, just tried the latest Sublime3 release (3080) on Win x64 and the Haxe builds doesn't work anymore unfortunately. I get no messages at all in the output window, but in the status bar a message that the build succeeded (no output generated though).

I see in the changelog that some build modifications has been made: http://www.sublimetext.com/forum/viewtopic.php?f=2&t=17471

**Comment by ciscoheat** [comment]
Great! Is the update available already in package manager?

**Comment by clemos** [comment]
Yes, it should (Package Manager version = master)

**Comment by ciscoheat** [comment]
Thanks

**Comment by ciscoheat** [comment]
Sorry, not sure what you mean with Package Manager version = master?

**Comment by clemos** [comment]
Package Control should automatically upgrade to latest master.

**Comment by ciscoheat** [comment]
Ok, I didn't happen yet, but maybe later...?

**Comment by clemos** [comment]
Unfortunately I'm not 100% sure how Package Control works internally.
You may need to restart ST or "upgrade package" manually through the command panel...

**Comment by r3a1ay** [comment]
Should be available now.

**Comment by ciscoheat** [comment]
Yes, it was automatically upgraded now and works fine! Thanks for your work with this.

**Comment by ciscoheat** [comment]
I'm experiencing the same problem again, also on build 3083. This time the syntax highlighting disappeared, and after using "Set Syntax: Haxe" on one file and pressing Ctrl-B, no output is generated. It find all the hxml files however, giving me a selection, but same for all.

Tried it on multiple projects and on Flashdevelop, so there's nothing wrong with the installation or compilation server. One or more haxe.exe processes are still hanging after I close Sublime.

I have cloned this repo and installed it manually, so let me know if I can assist in debugging.
